libs/helm/sidebar/src/lib/hlm-sidebar-wrapper.ts

```diff
@@ -1,24 +1,29 @@
 import { Directive, ElementRef, computed, inject, input } from '@angular/core';
 import { hlm } from '@spartan-ng/brain/core';
-
 import type { ClassValue } from 'clsx';
+import { injectHlmSidebarConfig } from './hlm-sidebar.token';
 
 @Directive({
 	selector: 'div[hlmSidebarWrapper]', // Restrict to main elements only
-
 	host: {
 		'data-slot': 'sidebar-wrapper',
 		'[class]': '_computedClass()',
+		'[style.--sidebar-width]': 'sidebarWidth()',
+		'[style.--sidebar-width-icon]': 'sidebarWidthIcon()',
```

Comment:
Moved the definition of the variables into the wrapper, like with shadcn/ui in `SidebarProvider` and added a config object.